clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/screenshare/services/IScreenshareConnection.as

```diff
@@ -0,0 +1,17 @@
+package org.bigbluebutton.lib.screenshare.services
+{
+	import flash.net.NetConnection;
+	
+	import org.osflash.signals.ISignal;
+
+	public interface IScreenshareConnection

```

Comment:
Duplicate class. This one can be removed.

Reply:
Deleted.